contracts/SupplyChain.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity 0.5.0;
+
+import "./Package.sol";
+
+
+contract SupplyChain {
+
+    mapping(bytes => Package) private packages;
```

Comment:
Why packageId is not bytes32 hash?

Reply:
I did not want to impose any limitations on what a `packageId` might actually be. I know `bytes32` would be cheaper to use but it's simpler this way and I think it's good enough for the workshops.